.travis.yml

```diff
@@ -125,7 +125,7 @@ install:
       if [ "$DO_COVERAGE" = "codecov" ]; then
         test -f build/luacov.stats.out || { echo 'build/luacov.stats.out does not exist.'; return 1; }
         luacov || return 1
-        travis_retry bash /tmp/codecov-bash -X gcov -X coveragepy -c -F "$1" || return 1
+        travis_retry bash /tmp/codecov-bash -X gcov -X coveragepy -c -F "$1,${LUANAME//./}" || return 1
```

Comment:
Why remove the dot? Does codecov choke on it? It took me a moment to figure out what his actually does, but basically this is fine with me...

Reply:
Yes.